airflow/utils/sqlalchemy.py

```diff
@@ -118,3 +120,27 @@ def process_result_value(self, value, dialect):
             type_map = {key.__name__: key for key in self.attr_keys}
             return type_map[data['type']](**data['attrs'])
         return data
+
+
+def skip_locked(session: Session, of: Any = None) -> Dict[str, Any]:
```

Comment:
```suggestion
def skip_locked(session: Session, of: Any = None) -> Dict[str, Any]:
```
Can you describe the `of` or use something more informative? I'm not sure if I got it 😄

Reply:
I wasn't aware of any more concrete type -- it's "any SQLA Model class". I'll see what I can find.

Reply:
My concern is not about typing but about the name of argument, the `of` is not meaningful to me

Reply:
Oh, right. That is mirroring the SQLA argument https://docs.sqlalchemy.org/en/13/orm/query.html?highlight=with_for_update#sqlalchemy.orm.query.Query.with_for_update:

> sqlalchemy.orm.query.Query.with_for_update(read=False, nowait=False, of=None, skip_locked=False, key_share=False)

Reply:
Oh wait, I see a way to remove this option from here:

`.with_for_update(**skip_locked(of=DagRun, session=session)` -> `with_for_update(of=DagRun, **skip_locked(session))`

Reply:
Fixed in https://github.com/apache/airflow/pull/10949/commits/3f4b2ea0c45541f462ad102646d685a74416aba2